packages/authentication/package.json

```diff
@@ -32,14 +37,12 @@
     "passport-strategy": "^1.0.0"
   },
   "devDependencies": {
+    "@loopback/build": "^4.0.0-alpha.1",
     "@loopback/openapi-spec-builder": "^4.0.0-alpha.8",
     "@loopback/testlab": "^4.0.0-alpha.10",
     "passport-http": "^0.3.0"
   },
-  "keywords": [
-    "LoopBack",
-    "Authentication"
-  ],
+  "keywords": ["LoopBack", "Authentication"],
```

Comment:
FWIW, we are not (or should not be) applying prettier formatting to `package.json`, because `npm` will reformat the file back to `JSON.stringify` format whenever we change it, typically via `npm install`. Could you please revert these white-space only changes?

I am not sure if `package.json` is listed in `.prettierignore`, feel free to add it there if it isn't. Here is what we did in loopback4-extension-starter (the trick works well with VS Code prettier plugin): 

https://github.com/strongloop/loopback4-extension-starter/blob/3a491d40e7d58ad177484de459050ba127215af7/.prettierignore#L4

```
*.json
```